hosted_control_planes/hcp-manage/hcp-manage-virt.adoc

```diff
@@ -20,25 +20,22 @@ The following table lists the capabilities that the infrastructure must provide
 
 .Persistent storage modes in a hosted cluster
 |===
-| Infrastructure CSI provider | Hosted cluster CSI provider | Hosted cluster capabilities | Notes
+| Infrastructure CSI provider | Hosted cluster CSI provider | Hosted cluster capabilities
 
 | Any RWX `Block` CSI provider
 | `kubevirt-csi`
 | Basic: RWO `Block` and `File`, RWX `Block` and `Snapshot`
-| Recommended
 
 | Any RWX `Block` CSI provider
-| {rh-storage-first} external mode
-| {rh-storage-first} feature set
-|
-
-| Any RWX `Block` CSI provider
-| {rh-storage-first} internal mode
-| {rh-storage-first} feature set
-| Do not use
-
+| {rh-storage-first}
+| {rh-storage-first} feature set. External mode has a smaller footprint and uses a standalone Red{nbsp}Hat Ceph Storage. Internal mode has a larger footprint, but is self-contained and suitable for use cases that require expanded capabilities such as RWX `File`.
 |===
 
+[NOTE]
+====
+In {hcp} on {VirtProductName}, {VirtProductName} handles storage on hosted clusters, which especially helps customers whose requirements are limited to block storage.
```

Comment:
```suggestion
In {hcp}, {VirtProductName} handles storage on hosted clusters, which especially helps customers whose requirements are limited to block storage.
```
Wondering if you could remove the first one since the entire section applies to OpenShift Virtualization so it's kind of redundant, rather than having OpenShift Virtualization repeat here. Just a thought.

Reply:
(it might not be a good suggestion - ignore if not!)